### PR TITLE
done settup eslint and prettier

### PR DESCRIPTION
**Pull request by NhatNguyen1502**

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "endOfLine": "crlf",
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "all",
```
Comment (khoanguyen-st) [inline .prettierrc.json]: Blocking: `trailingComma` should be `none`. We don't need the redundant comma at the end of array or object

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "endOfLine": "crlf",
+  "semi": true,
```
Comment (khoanguyen-st) [inline .prettierrc.json]: Blocking: `semi` should be `false`. We don't need the semicolon at the end of the line.

File: .prettierrc.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "jsxBracketSameLine": true,
```
Comment (khoanguyen-st) [inline .prettierrc.json]: Blocking: This seems to be deprecated. Pls remove it

File: .prettierrc.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  "bracketSameLine": true,
```
Comment (khoanguyen-st) [inline .prettierrc.json]: Blocking: Change to `false` for better readable

**Review by khoanguyen-st — APPROVED** [review]

**Review by vthu191 — APPROVED** [review]

**Review by ZietMy — APPROVED** [review]

**Pull request merged**